### ISSUE TITLE
Redirect version /2.10/ to /2/

### ISSUE THREAD

**Issue by NickVolynkin** [tarantool/installer.sh#24]
Apparently, a lot of people are trying to install from 
```
 curl -L https://tarantool.io/code/release/2.10/installer.sh | bash 
```

while the right URL is
```
 curl -L https://tarantool.io/code/release/2/installer.sh | bash 
```

**Comment by ylobankov** [comment]
> When we'll release 2.11 it'll install 2.11. Isn't it better to give an error message, which suggests the right URL?

Yes, I think it is much better.

**Comment by ylobankov** [comment]
> When we'll release 2.11 it'll install 2.11. Isn't it better to give an error message, which suggests the right URL?

Looks like it already gives an error.

```
[root@4ddbde830f73 /]# curl -L https://tarantool.io/code/release/2.10/installer.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   169  100   169    0     0   4423      0 --:--:-- --:--:-- --:--:--  4447
100   111  100   111    0     0   1033      0 --:--:-- --:--:-- --:--:--  1033
Tarantool version 2.10 not found
Available versions: 1.10, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 2, 3
[root@4ddbde830f73 /]#
[root@4ddbde830f73 /]# echo $?
0
```

But exit code is 0. I think it should be 1.

**Comment by foxzi** [comment]
fixed in https://github.com/tarantool/tarantool.io/pull/1509
